### PR TITLE
Adding member JSON data

### PR DESCRIPTION
**Pull request by bamblebam**

### CODE REVIEW

**Review by CodingSpecies — APPROVED** [review]
LGTM, thank you for contributing @bamblebam

File: public/data/bamblebam.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        "icon": "github"
+      },
+      {
+        "name": "Follow me on Twitter",
+        "url": "https://twitter.com/bamble_bam",
+        "icon": "twitter"
+      }
+    ]
+  }
+
```
Comment (Kunal0007) [inline public/data/bamblebam.json:18]: Please delete the last empty line

**Review by Kunal0007 — CHANGES_REQUESTED** [review]

File: package.json [hunk 2]
```diff
@@ -65,4 +65,4 @@
   "cypress-cucumber-preprocessor": {
     "nonGlobalStepDefinitions": true
   }
-}
\ No newline at end of file
+}
```
Comment (rohan-kulkarni-25) [inline package.json:68]: - Please don't commit package.json

File: public/data/bamblebam.json [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      "url": "https://github.com/bamblebam",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/bamble_bam",
+      "icon": "twitter"
+    }
+  ]
+}
```
Comment (Panquesito7) [inline public/data/bamblebam.json:17]:
```suggestion
}

```

File: public/data/bamblebam.json [hunk 4]
```diff
@@ -0,0 +1,17 @@
+{
+  "name": "Burhanuddin Rangwala",
+  "bio": "Web Developer, Machine Learning Engineer, ML-OPS, Web3, FreeLancer",
```
Comment (Panquesito7) [inline public/data/bamblebam.json:3]:
```suggestion
  "bio": "Web Developer, Machine Learning Engineer, ML-OPS, Web3, and Freelancer",
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Review by krishguptadev — APPROVED** [review]
LGTM

**Pull request merged**